test/common/autoscaling.go

```diff
@@ -0,0 +1,98 @@
+package common
+
+import (
+	"context"
+	rhmiv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/addon"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"time"
+)
+
+const (
+	hpaQuotaParam = "500"
+	hpaQuotaName  = "50 Million"
+)
+
+func TestAutoscaling(t TestingTB, ctx *TestingContext) {
+	quotaConfig, err := getQuotaConfig(t, ctx.Client)
+	if err != nil {
+		t.Fatalf("Error retrieving Quota: %v", err)
+	}
+
+	installation, err := GetRHMI(ctx.Client, true)
+	if err != nil {
+		t.Fatal("couldn't get RHMI cr for quota test")
+	}
+	if installation == nil {
```

Comment:
hey @bryankeane0 I don't think we need this check. It should be covered by getRHMI